kpi/tests/api/v2/test_api_submissions.py

```diff
@@ -248,45 +248,37 @@ def test_list_submissions_authenticated_asset_publicly_shared(self):
     def test_list_submissions_asset_publicly_shared_and_shared_with_user(self):
         """
         Running through behaviour described in issue kpi/#2870 where an asset
```

Comment:
@joshuaberetta @noliveleger I don't think GitHub ever automatically hyperlinks issues in code, but I try to stick with the format that it would link if it were in a description, comment, or commit message:
* If it's a reference to an issue in the same repository, just use `#1234`
* If it's a reference to an issue in a different repository, use `organization/repo#1234`

Example:
https://github.com/kobotoolbox/kpi/blob/90fd6c679d11171a792e06ec084c298428a14ae5/kpi/deployment_backends/base_backend.py#L29-L32